bot_cli/__main__.py

```diff
@@ -36,6 +42,18 @@ def main(address_book):
                 print_info("How can I help you?")
             case "add-contact":
                 commands.add_contact(args)
+            case "add-note":
+                commands.add_note(notebook, args)
+            case "update-note":
+                commands.update_note(notebook, args)
+            case "replace-note":
+                commands.replace_note(notebook, args)
```

Comment:
The functionality of your `replace_note` is more like what I would imagine under `change_note`. `update_note` looks like `add_text` I would discuss it with the team today on the meeting. But we cannot keep it like this.

Reply:
delete